src/clients/twitch.ts

```diff
@@ -0,0 +1,150 @@
+import { conjoinedMsg, twitchMsg } from '../messageObjects';
+import { authenticateTwitch } from '../oauth';
+import { Message } from 'discord.js';
+import bridge, { genericPromiseError } from './bridge';
+import configFile from '../../config.json';
+import tmijs from 'tmi.js';
+import fs from 'fs';
+
+let lastUserStateMsg: any = bridge.lastUserStateMsg;
```

Comment:
Ditto [this](https://github.com/iSlammedMyKindle/twitchToDiscordBot/pull/5/files#r957963819)